### ISSUE TITLE
missing @types/pushbullet

### ISSUE THREAD

**Issue by papakai** [alexwhitman/node-pushbullet-api#36]
to be able to work with pushbullet in an typescript development process, @types/pushbullet package needs to be available. Are you planning to dive into this in near future?

**Comment by alexwhitman** [comment]
I don't use Typescript myself so I haven't been planning on it.  I'm happy to help out if someone wants to take the lead on it.

**Comment by joekiller** [comment]
I didn't finish it, but here's a stab at 3.0 https://gist.github.com/joekiller/8e32d68be579532c78d215a2b54f53cc

**Comment by KaiSchwarz-cnic** [comment]
Find [here](https://www.newline.co/@bespoyasov/how-to-declare-missing-types-for-external-libraries--ff45540d#custom-types-declaration) also a quick way on how to just let the error disappear.

**Comment by joekiller** [comment]
Agreed. I have the "declare module" and many of the types completed in the gist just as the article suggests.
